.mega-linter.yml

```
@@ -0,0 +1,65 @@
+  - JSON_PRETTIER
+  - YAML_PRETTIER
+
+# ESLint configuration for TypeScript
+TYPESCRIPT_ES_CONFIG_FILE: .eslintrc.json
+TYPESCRIPT_ES_RULES_PATH: .
+
+# HTMLHint configuration - will use default rules if .htmlhintrc doesn't exist
+HTML_HTMLHINT_CONFIG_FILE: .htmlhintrc
+HTML_HTMLHINT_DISABLE_ERRORS_IF_LESS_THAN: 0
+
+# Stylelint configuration - will use default rules if config doesn't exist  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)